src/Core/tests/UnitTests/ElementHandlerTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Core.UnitTests.TestClasses;
+using Microsoft.Maui.UnitTests;
+using Xunit;
+
+namespace Core.UnitTests
+{
+
+	[Category(TestCategory.Core)]
+	public class ElementHandlerTests
+	{
+		[Fact]
+		public void CreatePlatformElementUsingPlatformElementFactory()
+		{
+			ElementHandlerStub.PlatformElementFactory = handler => new CustomPlatformCell();
+
+			var elementStub = new ElementHandlerStub();
+			elementStub.SetVirtualView(new Microsoft.Maui.Controls.TextCell());
+
+			Assert.NotNull(elementStub.PlatformView);
+			Assert.True(elementStub.PlatformView is CustomPlatformCell);
```

Comment:
I think this is better for error messages and will fail if it is not exactly that type:

```suggestion
			Assert.IsType<CustomPlatformCell>(elementStub.PlatformView);
```